cellprofiler/gui/pipelinecontroller.py

```diff
@@ -133,6 +133,8 @@ def __init__(self, workspace, frame):
         wx.EVT_MENU(frame, cpframe.ID_EDIT_DELETE, self.on_remove_module)
         wx.EVT_MENU(frame, cpframe.ID_EDIT_DUPLICATE, self.on_duplicate_module)
 
+        wx.EVT_MENU(frame, cpframe.ID_EDIT_BROWSE_FOR_FOLDER,
+                    self.on_pathlist_browse_folder)
         wx.EVT_MENU(frame, cpframe.ID_EDIT_BROWSE_FOR_FILES,
                     self.on_pathlist_browse)
```

Comment:
I would actually modify the original `on_pathlist_browse` to be `on_pathlist_browse_images`, just to make a better distinction from `on_pathlist_browse_folder`.